### PR TITLE
Saving file size

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSBPS-827

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/BlobProcessor.java [hunk 1]
```diff
@@ -70,7 +70,8 @@ public void process(BlobClient blobClient) {
             envelopeSupplier = () -> envelopeService.createNewEnvelope(
                 blobClient.getContainerName(),
                 blobClient.getBlobName(),
-                blobClient.getProperties().getCreationTime().toInstant()
+                blobClient.getProperties().getCreationTime().toInstant(),
+                blobClient.getProperties().getBlobSize()
```
Comment (mdayican) [inline src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/BlobProcessor.java:74]: which one is correct `blobClient.getProperties().getBlobSize()` or `b.getProperties().getContentLength()` or

File: src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/DuplicateFinder.java [hunk 2]
```diff
@@ -29,7 +29,12 @@ public DuplicateFinder(
             .listBlobs()
             .stream()
             .filter(b -> isDuplicate(b.getName(), containerName))
-            .map(b -> new Duplicate(b.getName(), containerName, b.getProperties().getLastModified().toInstant()))
+            .map(b -> new Duplicate(
+                    b.getName(),
+                    containerName,
+                    b.getProperties().getLastModified().toInstant(),
+                    b.getProperties().getContentLength()
```
Comment (mdayican) [inline src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/DuplicateFinder.java:36]: which one is correct `blobClient.getProperties().getBlobSize()` or `b.getProperties().getContentLength()`

File: src/test/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/BlobProcessorTest.java [hunk 3]
```diff
@@ -285,7 +286,8 @@ private void verifyNewEnvelopeHasBeenCreated() {
         verify(envelopeService).createNewEnvelope(
             blobClient.getContainerName(),
             blobClient.getBlobName(),
-            blobClient.getProperties().getCreationTime().toInstant()
+            blobClient.getProperties().getCreationTime().toInstant(),
+            blobClient.getProperties().getBlobSize()
```
Comment (mdayican) [inline src/test/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/BlobProcessorTest.java:290]: which one is correct `blobClient.getProperties().getBlobSize()` or `b.getProperties().getContentLength()`

Comment (apereverzin) [reply src/test/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/BlobProcessorTest.java:290]: I assume they are interchangeable:

- in DuplicateFinder BlobItem class is used (`b.getProperties().getContentLength()`)
- in BlobProcessor BlobClient class is used (`blobClient.getProperties().getBlobSize()`)

`blobClient` is created in our ContainerProcessor class like this:
`containerClient.getBlobClient(blobItem.getName())`

I will rename `b` to `blobItem` in `DuplicateFinder` for better clarity.

**Pull request merged**